packages/prismic-cli/src/prismic/base-command.ts

```diff
@@ -130,6 +130,7 @@ export default abstract class PrismicCommand extends Command {
     if (this.isAbsoluteUrlToZip(source)) return source
     if (this.isGithubUrl(source) === false) return Promise.reject(new Error(`Could not guess where to find zip from ${source}`))
     const url = new URL(source)
+    url.pathname = url.pathname.replace(/.git$/, '')
```

Comment:
redondant with #126 or should be based on `mm/themes-ending-with-dot-git` to prevent this part of the diff.